db/migrate/20150325224103_add_type_to_project_account.rb

```diff
@@ -0,0 +1,10 @@
+class AddTypeToProjectAccount < ActiveRecord::Migration
+  def change
+    add_column :project_accounts, :person_type, :text
+
+    execute <<-SQL
+      update project_accounts set person_type  = 'Pessoa Física' where state_inscription  is null; 
+      update project_accounts set person_type  = 'Pessoa Jurídica' where state_inscription  is not null; 
```

Comment:
I would change this name. Person type does not make much sense.
Besides, is completely dependent (therefore can be derived from) CPF/CNPJ.

Why we don't simply use a method to infer this from the document used?